app/presenters/hyku/work_show_presenter.rb

```diff
@@ -53,6 +55,28 @@ def user_can_feature_collection?
     end
     # End Featured Collections Methods
 
+    # @return [Boolean] render a IIIF viewer
+    def iiif_viewer?
+      representative_id.present? &&
+        representative_presenter.present? &&
+        iiif_media? &&
+        Hyrax.config.iiif_image_server? &&
```

Comment:
Move this line to the top of the list; it's the fastest to evaluate.  We want to have the quickest ones at the beginning of the `&&` chain.